util/settings/filtered_resource.go

```diff
@@ -5,13 +5,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
-type ExcludedResource struct {
+type FilteredResource struct {
```

Comment:
I feel like you might want to call this `ResourceFilter`?

Reply:
I actually think `FilteredResource` is more appropriate. In this case it reads as a "resource that is filtered" while `ResourceFilter` reads more like "a filter of resources". Since the type/struct refers to a resource and not the filter I think what I have is more appropriate. This is a tad pedantic though, so let me know what you think!